packages/core-api/src/extensions/extensions.tsx

```diff
@@ -20,7 +20,7 @@ import { BackstagePlugin, Extension } from '../plugin/types';
 import { RouteRef, useRouteRef } from '../routing';
 import { attachComponentData } from './componentData';
 
-type ComponentLoader<T> =
+export type ComponentLoader<T> =
```

Comment:
I'll have a look at refactoring these to see if we can make this unnecessary.

Reply:
We actually don't need this I think - Just went with sync components for now. Let's see if we 100% need it.